webrtc-stats.html

```diff
@@ -1256,6 +1258,39 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>estimatedPlayoutTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated playout time of this track.
+                  The playout time is the NTP timestamp of the last played out audio sample or video
+                  frame that has a known timestamp, extrapolated with the time elapsed since it was
+                  played out. This is the "current time" of the track and can be present even if
+                  there is no audio or video currently playing.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>estimatedCaptureTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated capture time of this track.
+                  The capture time is the NTP timestamp of the last played out audio sample or video
+                  frame that has a known timestamp + half of what the estimated round trip time was
+                  when that audio sample or video frame arrived, extrapolated with the time elapsed
```

Comment:
Do we really need an RTT estimate? 

Because the receiver gets a sync event (RTCP SR containing the RTP timestamp and the corresponding UNIX timestamp. We could essentially use the time elapsed in the local clock to estimate the captureTimestamp.